repair_purchase_return/models/purchase_return_order.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (C) 2021 ForgeFlow S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html)
+
+from odoo import fields, models
+
+
+class PurchaseReturnOrder(models.Model):
+    _inherit = "purchase.return.order"
+
+    repair_order_count = fields.Integer(
+        compute="_compute_repair_orders",
+        string="Repair Order Count",
+        copy=False,
+        default=0,
+    )
+    repair_order_ids = fields.Many2many(
+        comodel_name="repair.order",
+        compute="_compute_repair_orders",
+        copy=False,
```

Comment:
@ChrisOForgeFlow  does this need review to merge? 
```
purchase.return.order: inconsistent 'compute_sudo' for computed fields: repair_order_count, repair_order_ids
```
do we need to override the compute_sudo to False when adding compute?
```suggestion
        copy=False,
        compute_sudo=False,
```

Reply:
The reason for this issue, is related with definition of computed field, two field have same computed method, but one is stored and another not, because that is the issue, or all are stored, or all are not stored if share method to compute